### DISCUSSION TITLE
2024/11/25/%E5%A6%82%E4%BD%95%E6%9F%A5%E8%AF%A2repo-id%E5%92%8Ccategory-id/

### DISCUSSION THREAD

**Discussion by giscus** [moyigeek/blog#5] [Announcements]
# 2024/11/25/%E5%A6%82%E4%BD%95%E6%9F%A5%E8%AF%A2repo-id%E5%92%8Ccategory-id/

如何查询repo_id和category_id在配置基于 Github Discussions的博客评论系统时(例如aiscus)，往往需要获取repo的repoId、cateqoryId 等属性，因此这里介绍一种获这些信息的方法。 首先需要开启仓库的 Discussions 功能。在仓库的 Settings -> Options -> Features 中找到 Discussions

https://blog.moyihust.eu.org/2024/11/25/%E5%A6%82%E4%BD%95%E6%9F%A5%E8%AF%A2repo-id%E5%92%8Ccategory-id/

**Comment by thethepai** [comment]
学到了
